### ISSUE TITLE
Unable To Import Scoped Styles

### ISSUE THREAD

**Issue by shadesOfcodes** [cmseguin/rollup-plugin-react-scoped-css#28]
For some weird Reasons, Rollup Crashes whenever i try to import the scoped scss, says file not found, and the file path always ends with a query like ```Gallery\gallery.scoped.scss?scope=f4b9bc0b```

**Comment by cmseguin** [comment]
Can you share your configs, I have to be honest, I haven't got as far as doing test on projects that are not using vite.

**Comment by shadesOfcodes** [comment]
```const reactScopedPlugins = reactScopedCssPlugin();

export default {
  input: "src/index.js",
  output: [
    {
      file: pkg.main,
      format: "cjs",
      sourcemap: true,
    },
    {
      file: pkg.module,
      format: "es",
      sourcemap: true,
    },
  ],
  plugins: [
    reactScopedPlugins[0],
    external(),
    postcss({
      modules: false,
    }),
    url(),
    babel({
      exclude: "node_modules/**",
      plugins: ["external-helpers"],
    }),
    resolve(),
    commonjs(),
    reactScopedPlugins[1],
  ],
};
```

**Comment by cmseguin** [comment]
I will try work on replicating your project in the examples tonight and will try to work on a fix ASAP. Feel free to open a PR if you want to setup the example repo so we can better resolve this issue! :)

**Comment by shadesOfcodes** [comment]
> I will try work on replicating your project in the examples tonight and will try to work on a fix ASAP. Feel free to open a PR if you want to setup the example repo so we can better resolve this issue! :)
> 
> Is your project using JSX in any form?

without jsx, this error doesn't come up and scoped css doesn't seem to work...

I am currently trying to modify this project [react-shimmer-effects](https://github.com/shafikulz/react-shimmer-effects). The styles used in this project were causing conflicts  with tailwinds in  a project i am currently using it on. so i am trying to fix the issue by scoping the styles used

**Comment by cmseguin** [comment]
Thank you! I didn't really have a usecase to use react without vite but this is a great example !👍  I will try to work on it ASAP and find a solution!

**Comment by cmseguin** [comment]
I am trying to install the example and can't even install the dependency from react-shimmer-effects. Also one thing I noticed is that the react file are in .js extension so for sure you would have to add options to specify your jsx is located in .js files or convert your files to .jsx

```js
const reactScopedPlugins = reactScopedCssPlugin({
  jsxFileExtensions: ['js', 'jsx', 'ts', 'tsx']
})
```

**Comment by shadesOfcodes** [comment]
To Install The Dependency ensure to use npm, with package.lock files intact, some packages used are not compatible with the most recent versions on react eg  react script.

**Comment by shadesOfcodes** [comment]
let me set up a stable example copy of the react-shimmer for you to work it,

**Comment by shadesOfcodes** [comment]
> I am trying to install the example and can't even install the dependency from react-shimmer-effects. Also one thing I noticed is that the react file are in .js extension so for sure you would have to add options to specify your jsx is located in .js files or convert your files to .jsx
> 
> ```js
> const reactScopedPlugins = reactScopedCssPlugin({
>   jsxFileExtensions: ['js', 'jsx', 'ts', 'tsx']
> })
> ```

yeah i had to change some of the files to jsx extensions,  will probably use the above approach later, and then i realised that moving the second part of the react-scoped-plugin just below the postcss plugin, in the rollup config file, fixed the stylesheet not found issue, but reactScopedCssPlugin, started throwing some jsx related syntax errror.

**Comment by shadesOfcodes** [comment]
Oh, I had to find an alternative to react-shimmer-effects, will try the new version out later and see if it fixes it

**Comment by shaniqwa** [comment]
I have the same issue. any updates on this? tnx
